torch/csrc/autograd/python_variable.cpp

```diff
@@ -112,6 +121,15 @@ static const char* VOLATILE_WARNING =
     "volatile was removed and now has no effect. Use "
     "`with torch.no_grad():` instead.";
 
+bool check_has_torch_dispatch(PyObject *obj) {
```

Comment:
nit: `static bool check_has_torch_dispatch`, unless you wanted to toss this into a header?